### ISSUE TITLE
scripts/net/net-location.awk: hash

### ISSUE THREAD

**Issue by cup** [cup/bay#22]
~~~
unix.stackexchange.com/a/346069
unix.stackexchange.com/q/346050
unix.stackexchange.com/q/346050#346069
unix.stackexchange.com/questions/346050
unix.stackexchange.com/questions/346050/fix-config-in-vimrc-file
unix.stackexchange.com/questions/346050/fix-config-in-vimrc-file/346069#346069
~~~

http://unix.stackexchange.com/q/453753/curl-url-effective-with-hash

**Comment by svnpenn** [comment]
https://github.com/svnpenn/velour/issues/139

**Comment by svnpenn** [comment]
https://github.com/svnpenn/velour/issues/153

**Comment by cup** [comment]
we dont need this

any hashes for original URLs (read: no home baked URLs) are not needed - are they will be duplicated in the URL:

https://unix.stackexchange.com/questions/346050/fix-config-in-vimrc-file/346069#346069

so what we need to do i just go through and remove any hash - as they are not needed

then we can finish the script - if its not finished already
